### ISSUE TITLE
Convert .textile to .md

### ISSUE THREAD

**Issue by mjpost** [naacl-org/naacl-org.github.com#2]
We are currently using .textile to process many of our pages. This needs to be [switched over to Markdown since Github is retiring .textile on May 1](https://help.github.com/articles/updating-your-markdown-processor-to-kramdown/).

**Comment by cherryc** [comment]
Well, we ignored this and our website didn't blow up, so that's good news :-)

I'm going to take a stab at this, unless someone wants to jump on and say it would be trivial for them. My strategy will be to use [https://github.com/jgm/pandoc](pandoc).

**Comment by mjpost** [comment]
I think the next time we push it will fail to build.

**Comment by cherryc** [comment]
Is it broken for anyone else now? I was working under the same assumption as Matt, that we needed to fix it before the next push. But I haven't pushed anything, and now the front page is slow for me and all other pages just don't work.

**Comment by mjpost** [comment]
Yes, I'm also having trouble loading the main page (and it doesn't seem to be a problem with Github itself).

**Comment by mjpost** [comment]
Yes, I'm also having trouble loading the main page (and it doesn't seem to be a problem with Github itself).

**Comment by mjpost** [comment]
Yes, I'm also having trouble loading the main page (and it doesn't seem to be a problem with Github itself).

**Comment by cherryc** [comment]
Okay - fortunately, I think I have a solution almost ready.

**Comment by mjpost** [comment]
The page does load for me eventually, but nearly unformatted. Thanks for taking this on!

**Comment by cherryc** [comment]
Okay, I've checked in a script that I've run to replace all .textile with .md. It displays fine on my machine, but doesn't seem to make a difference when it comes to fixing our website. I wonder if this issue was unrelated? I'm not sure how we check.

**Comment by mjpost** [comment]
I am starting to think it's unrelated:

- I created this issue because a commit triggered an email. I haven't gotten any emails from your current commit, which should happen if there was a jekyll build fail.

- I can "wget naacl.org/index.html" without any trouble, suggesting that it's one of the recursive fetches that is causing the problem.

That's all I've looked into it.

**Comment by cherryc** [comment]
Okay, I'm giving up for now. Like I said, the local build continues to work fine, and I agree that we aren't getting any build errors from the server, so I'm stumped.

**Comment by cherryc** [comment]
I've fixed it - something was going wrong with "baseurl" and our redirect. Basically, naacl-org.github.com no longer works, but naacl.org works fine. The problem is that all our web addresses in the site's HTML and markdown were pointing to "baseurl" which was pointing to naacl-org.github.com. I changed "baseurl" to point be "naacl.org" and everything seems to be working fine.

So, it was unrelated to the markdown issue. I kind of wonder if we can get away without any baseurl at all. I may try that later today.
